src/main/java/com/minecolonies/core/colony/ColonyManager.java

```diff
@@ -327,13 +327,22 @@ public boolean isFarEnoughFromColonies(@NotNull final Level w, @NotNull final Bl
     @NotNull
     public List<IColony> getColonies(@NotNull final Level w)
     {
-        final IColonyManagerCapability cap = w.getCapability(COLONY_MANAGER_CAP, null).resolve().orElse(null);
-        if (cap == null)
+        if (w.isClientSide())
```

Comment:
we have a specific function to get an IColony already, should make a new function in the same style instead. Else it'd be confusing that this one does return client colonies but all the other calls do not

Reply:
This is that function though.  It just stopped working on client side at some point.

tbf the whole colony API is messy, with `IColonyManager` having some single-sided methods and some both-sided methods, and `IColony` being both-sided some of the time and not all of the time.  But this is where we are right now.

Reply:
the specific both sided function is called
 `public IColony getIColony(@NotNull final Level w, @NotNull final BlockPos pos)` 
 Instead of modifying this serverside used one, potentially introducing hard to track sidededness bugs I want an extra new method instead called 
 public List<IColony> getIColonies(@NotNull final Level w)
 which follows the same style